### ISSUE TITLE
Lack of support for person_* emojis

### ISSUE THREAD

**Issue by hairmare** [mattermost/mattermost-server#17328]
#### Summary
Non-binary variants of popular emojis like faceplaming are not supported.

Affects all the emojis available as `:man_*` and `:women_*`, I'm using `:person_facepalming:` as a somewhat on-topic example 😉 

#### Steps to reproduce
Use non-binary emojis like 🤦

I tend to insert them via OS native keyboard but if I would pseudocode them as MM-markup I would do this:

```
# :person_faceplaming: :man_facepalming: :woman_facepalming:
```

#### Expected behavior
The emojis should be rendered in a consistent fashion.

#### Observed behavior (that appears unintentional) 
The `:person_facepalming:` emoji isn't getting rendered by MM and for some reason the Unicode glyph doesn't get formatted as a "title" due to `#`. (the `#` is not this issue, it's just a visual aid)

![image](https://user-images.githubusercontent.com/116588/113348960-99568280-9337-11eb-840f-5a1223abff21.png)

#### Possible fixes
Add it to `model/emoji_data.go`.

 I can try and contribute a fix but it feels like something that the right maintainer can fix way faster than me as a drive-by contributor.

**Comment by amyblais** [comment]
@hmhealey Is this part of the Emoji v13.1 upgrade effort we're planning for Q2, or would this be a separate ticket?

**Comment by bandisast** [comment]
I believe these gender-neutral emojis were added in Unicode 12.1, so they should already be included in the Emoji v13.1 effort. 
Loose source: https://blog.emojipedia.org/unicode-brings-forward-gender-neutral-timeline/

**Comment by hmhealey** [comment]
Yep, this will be added with Emoji 13.0. We're currently way out of date with our emoji support (only including Emoji 4.0), and we're planning on fixing that in the next while.

FYI, @amyblais, but we're only planning on Emoji 13.0 right now instead of 13.1 since support for 13.1 is limited currently.
